### PR TITLE
Update Project & Travis settings for Xcode 10, Swift 4.2

### PR DESCRIPTION
**Pull request by jjgp**
Updating Travis, project settings for Xcode 10 and Swift 4.2.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,16 +10,16 @@ env:
     - FRAMEWORK_NAME="ReSwiftRouter"
     - UPDATE_DOCS="false"
 
-osx_image: xcode8.2
+osx_image: xcode10
```
Comment (jjgp) [inline .travis.yml]: @DivineDominion this should be all that is needed to get it to work with travis in your PR. I updated the project settings and Swift version if wanted.

File: ReSwiftRouter.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -224,8 +224,7 @@
 				25C0A9CB1C50AF6400139FA7 /* Router.swift */,
 				625E66E21C2000620027C288 /* Frameworks */,
 			);
-			name = ReSwiftRouter;
-			path = SwiftFlowRouter;
```
Comment (jjgp) [inline ReSwiftRouter.xcodeproj/project.pbxproj]: Group reference had SwiftFlowRouter and was showing red in the project. changed to the folder.

**Comment by jjgp** [comment]
https://github.com/ReSwift/ReSwift-Router/pull/102#issuecomment-427526950

Decreased because coverage hasn't been gathered since failing builds

**Comment by mjarvis** [comment]
Thanks @jjgp ! Like all contributors I'll invite you to the ReSwift org. That will make future contributions easier in case you are interested! 🎉

**Comment by jjgp** [comment]
Thanks @mjarvis I will gladly join

**Pull request merged**